package.json

```diff
@@ -20,11 +20,16 @@
   "scripts": {
     "format:check": "prettier --check .",
     "lint": "eslint --cache .",
-    "prepare": "husky"
+    "prepare": "husky",
+    "prepublishOnly": "babel src --out-dir src-cjs"
```

Comment:
The workflow defined [in this repo](https://github.com/bloq/commitlint-config-bloq/blob/master/.github/workflows/npm-publish.yml) doesn't call the `prepublishOnly` command manually (as it must be done when using `JS-DevTools/npm-publish@v3`

Reply:
Fixed in c621d64.